packages/jsii-dotnet-runtime/src/Amazon.JSII.Runtime/Services/NodeProcess.cs

```diff
@@ -48,5 +64,23 @@ void IDisposable.Dispose()
             StandardError.Dispose();
             _process.Dispose();
         }
+        
+        /// <summary>
+        /// Gets the target framework attribute value and
+        /// the assembly file version for the current .NET assembly
+        /// </summary>
+        /// <returns>A tuple where Item1 is the target framework
+        /// ie .NETStandard,Version=v2.0
+        /// and item2 is the assembly file version (ie 1.0.0.0)</returns>
+        private Tuple<string, string> GetAssemblyFileVersion()
+        {
+            var assembly = typeof(NodeProcess).GetTypeInfo().Assembly;
+            var assemblyFileVersionAttribute = assembly.GetCustomAttribute(typeof(AssemblyFileVersionAttribute)) as AssemblyFileVersionAttribute;
+            var frameworkAttribute = assembly.GetCustomAttribute(typeof(TargetFrameworkAttribute)) as TargetFrameworkAttribute;
+            return new Tuple<string, string>(
+                frameworkAttribute == null ? "Unknown" : frameworkAttribute.FrameworkName,
+                assemblyFileVersionAttribute == null ? "Unknown" : assemblyFileVersionAttribute.Version
+                );
```

Comment:
Is this bracket intended to be indented there? Looks to me it should be indented one less deep?

Reply:
Yeah, rider bug. Indentation is wrong when a new is in a return statement for some reason.

Fixed.